.github/pull_request_template.yml

```diff
@@ -0,0 +1,7 @@
+## Changes
+
+_What does this PR change? Link to any related issue(s)._
+
+## How to Review
+
+_How can a reviewer review your changes? What should be kept in mind for this review?_
```

Comment:
Note: I have been using this 2-question template for years and still like it. It keeps the PR description light, but helpful:

1. What am I doing?
2. What am I asking of others? How can I get the feedback I need?

IMO any additional information builds off these 2 pillars, so I always find myself falling back to these as the minimum (a checklist can help, too, if there are things that can’t be automated in CI).

Reply:
Yup, these two are good!